examples/test-coverage.yaml

```diff
@@ -31,7 +31,8 @@ jobs:
           covr::codecov(
             quiet = FALSE,
             clean = FALSE,
-            install_path = file.path(normalizePath(Sys.getenv("RUNNER_TEMP"), winslash = "/"), "package")
+            install_path = file.path(normalizePath(Sys.getenv("RUNNER_TEMP"), winslash = "/"), "package"),
+            token = Sys.getenv("CODECOV_TOKEN")
```

Comment:
This is not needed, covr picks up the token automatically.

Reply:
Yes, I know, I thought it might be better to explicitly pass it to the function to make the intent of using a CODECOV_TOKEN visible.
It will work without but I thought it might help other to understand the code.

Reply:
It also breaks the cases when the token is in the config file, because covr does this:
```
token <- token %||% Sys.getenv("CODECOV_TOKEN", extract_from_yaml(attr(coverage, "package")$path))
```
So if the env var is not set, then it reads the config file. But if you pass a `""` here, then it won't.

Which possible also means that we cannot set that env var unconditionally, because then if there is no secret, it'll set it to an empty string (probably), which is different than not being set.

Reply:
Alright, thanks for driving the point home.